Sources/MapboxMaps/Style/Generated/Layers/SymbolLayer.swift

```diff
@@ -99,8 +99,8 @@ public struct SymbolLayer: Layer {
       public var textAnchor: Value<TextAnchor>?
             
       /// Value to use for a text label. If a plain `string` is provided, it will be treated as a `formatted` with default/inherited formatting options.
-      public var textField: Value<String>?
-            
+      public var textField: Value<Formatted>? // This change will get permanently overwritten and when bindings are generated
```

Comment:
```suggestion
      public var textField: Value<Formatted>? // This change will get permanently overwritten when bindings are generated
```

what's our solution?

Reply:
@macdrevx see https://github.com/mapbox/mapbox-maps-ios/pull/242 as it updates bindings. I made a manual change so that tests will pass